catalogValidationValues.json.template

```diff
@@ -0,0 +1,5 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY,
+  "resource_group_name": $PREFIX,
+  "existing_kms_instance_guid": $HPCS_US_SOUTH_GUID
```

Comment:
I don't suggest using an existing instance here so the validation tests key protection creation